action.yml

```diff
@@ -15,6 +15,10 @@ inputs:
     required: true
     description: 'The action will skip PRs that have less approvals than this value'
     default: '2'
+  require_passed_checks:
+    required: false
+    description: "If the action should skip PRs that have failed checks, defaults to `true`"
+    default: "true"
```

Comment:
Can we use a boolean here instead of string?

Reply:
good catch, bool makes more sense I think.  
Although we can only get a string with `getInput` https://github.com/actions/toolkit/issues/361#issuecomment-590469142

Reply:
Do you mind changing this to a bool? @alejandrohdezma  Thanks
Or I can make the change after merging this PR

Reply:
[There is no difference for GitHub for using a boolean here or a string](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs), since it only uses strings, as @zhiyelee pointed out

<img width="686" alt="Screenshot 2021-04-05 at 19 29 22" src="https://user-images.githubusercontent.com/9027541/113604224-40dbf980-9645-11eb-96e2-984bf6bf9854.png">

Reply:
Also, [here](https://github.com/SchemaStore/schemastore/blob/d3d49a5e0a024088e161877b54500baba72b5c30/src/schemas/json/github-action.json#L290) is the Json Schema for GitHub actions

Reply:
makes sense. Thanks for the clarification